components/gantry/oneaxis/oneaxis.go

```diff
@@ -22,6 +22,7 @@ import (
 )
 
 var model = resource.DefaultModelFamily.WithModel("oneaxis")
+var LIMIT_ERROR_MARGIN = 0.25
```

Comment:
I think this can be a `const` rather than a `var`. I'd also put in a quick comment explaining that this is how far we'll let you move past where we expect the limit switch to be before we stop the movement.